src/customer-ontology.ttl

```diff
@@ -0,0 +1,122 @@
+  rdfs:isDefinedBy customer: ;
+  rdfs:isSubPropertyOf rdfs:label ;
+  rdfs:domain customer:Customer ;
+  rdfs:range xsd:string .
+
+customer:displayName a rdf:Property ;
+  rdf:label "displayName" ;
+  rdfs:comment "A literal string to be used when presenting an institution as text to users" ;
+  rdfs:isDefinedBy customer: ;
+  rdfs:isSubPropertyOf rdfs:label ;
+  rdfs:domain customer:Customer ;
+  rdfs:range xsd:string .
+
+customer:shortName a rdf:Property ;
+  rdf:label "shortName" ;
+  rdfs:comment "An abbreviation of an institution name" ;
+  rdfs:isDefinedBy customer: ;
+  rdfs:isSubPropertyOf rdfs:label ;
+  rdfs:domain customer:Customer ;
+  rdfs:range xsd:string .
+
+customer:archiveName a rdf:Property ;
+  rdf:label "archiveName" ;
+  rdfs:comment "A name used when describing the set of data owned by an institution" ;
+  rdfs:isDefinedBy customer: ;
+  rdfs:isSubPropertyOf rdfs:label ;
+  rdfs:domain customer:Customer ;
+  rdfs:range xsd:string .
+
+customer:cname a rdf:Property ;
+  rdfs:comment "A Canonical Name (DNS) that should resolve the set of data owned by an institution" ;
+  rdfs:isDefinedBy customer: ;
+  rdfs:domain customer:Customer ;
+  rdfs:range xsd:string .
+
+customer:institutionDns a rdf:Property ;
+  rdfs:comment "A Domain Name System used by an institution" ;
+  rdfs:isDefinedBy customer: ;
+  rdfs:domain customer:Customer ;
+  rdfs:range xsd:string .
+
+customer:feideOrganizationId a rdf:Property ;
+  rdfs:comment "A Feide organization ID of an institution" ;
+  rdfs:isDefinedBy customer: ;
+  rdfs:domain customer:Customer ;
+  rdfs:range xsd:string .
+
+customer:cristinId a rdf:Property ;
+  rdfs:comment "An identifier in the Cristin organization register associated with an institution" ;
+  rdfs:isDefinedBy customer: ;
+  rdfs:domain customer:Customer ;
+  rdfs:range xsd:string .
+
+customer:vocabularySetting a rdf:Property ;
+  rdfs:comment "A resource defining the usage of a controlled vocabulary" ;
+  rdfs:isDefinedBy customer: ;
+  rdfs:domain customer:Customer ;
+  rdfs:range customer:VocabularySetting .
+
+customer:name a rdf:Property ;
+  rdf:label "name" ;
+  rdfs:comment "A literal string by which a thing is known" ;
+  rdfs:isDefinedBy customer: ;
+  rdfs:isSubPropertyOf rdfs:label ;
+  rdfs:domain rdfs:Class ;
+  rdfs:range xsd:string .
+
+customer:vocabularyUsage a rdf:Property ;
+  rdfs:comment "A definition of the allowed usage of a controlled vocabulary" ;
+  rdfs:isDefinedBy customer: ;
+  rdfs:domain customer:VocabularySetting ;
+  rdfs:range customer:VocabularyStatus .
+
+customer:created a rdf:Property ;
+  rdfs:comment "A point in time when a resource was first created" ;
+  rdfs:isDefinedBy customer: ;
+  rdfs:domain rdfs:Class ;
+  rdfs:range xsd:dateTime ;
+  rdfs:subPropertyOf dcterms:created .
+
+customer:modified a rdf:Property ;
+  rdfs:comment "A point in time when a resource was last changed" ;
```

Comment:
Should the format of the dateTime be specified?

Reply:
rdfs:range xsd:dateTime ;

Reply:
Is that allways yyyy-mm-dd?

Reply:
http://books.xmlschemata.org/relaxng/ch19-77049.html

Reply:
Note that there may be an issue with time zones, but…I have to admit I am not sure